<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background: #262626;
	color: white;
	selection-background-color: #2cde85;
	selection-color: black;
}


QPushButton {
	border-radius: 5px;
	border: 1px solid white;
	color: white;
	min-width: 85px;
	min-height: 25px;
}

QPushButton#next_btn, #prev_btn {
		min-width: 60px;
}

QPushButton::hover {
	background: #2cde85;	
	color: black;
	border: none;

}

QFrame#frame {
	/*background-color: white;*/
	border: 1px solid white;
}


QLineEdit {
	min-height: 20px;
	border: 1px solid white;
}

QHeaderView::section {
	background-color: black;
	color: white;
}

QComboBox {
	color: white;
	border: 1px solid white;
}

QDateEdit {
	border: 1px solid white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>50</number>
   </property>
   <property name="leftMargin">
    <number>50</number>
   </property>
   <property name="topMargin">
    <number>50</number>
   </property>
   <property name="rightMargin">
    <number>50</number>
   </property>
   <property name="bottomMargin">
    <number>50</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>30</number>
     </property>
     <property name="verticalSpacing">
      <number>5</number>
     </property>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="btns_widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>140</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="prev_btn">
        <property name="minimumSize">
         <size>
          <width>62</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>34</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">max-width: 32px;</string>
        </property>
        <property name="text">
         <string>&lt;-</string>
        </property>
        <property name="icon">
         <iconset theme="go-previous">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="next_btn">
        <property name="minimumSize">
         <size>
          <width>62</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>34</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">max-width: 32px;</string>
        </property>
        <property name="text">
         <string>-&gt;</string>
        </property>
        <property name="icon">
         <iconset theme="go-previous">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save_btn">
        <property name="minimumSize">
         <size>
          <width>87</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
        <property name="icon">
         <iconset theme="go-previous">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="del_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>87</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
        <property name="icon">
         <iconset theme="go-previous">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="back_btn">
        <property name="minimumSize">
         <size>
          <width>87</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Закрыть</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
